code/mob/living/silicon/robot.dm

```diff
@@ -226,7 +226,6 @@
 			logTheThing("combat", src, null, "'s AI controlled cyborg body was destroyed [log_health(src)] at [log_loc(src)].") // Brought in line with carbon mobs (Convair880).
 			src.mainframe.return_to(src)
 		setdead(src)
-		borg_death_alert()
```

Comment:
?

Reply:
Completely forgot about that bug fix; in certain circumstances, the borg death alert proc can be called more than once, resulting in weird behavior with double PDA messages (extensive testing and headscratching with people in #ImCoder indicated that this was the most likely cause; deleting this line fixed the issue and HOPEFULLY won't fuck other things over)